OlinkAnalyze/cran-comments.md

```diff
@@ -1,6 +1,6 @@
 ## Release Summary
 
-This release contains extending the functionality of to existing functions (from version 3.6.2) to new data. This version also contains improvements to tutorials for clarity.
+This release contains new functionality and tutorials related to LOD.
```

Comment:
Optional change: should we specify that this is for Explore 3072 and Explore HT LOD?